apps/dashboard_app/helpers/loans_table.py

```diff
@@ -8,9 +11,25 @@
 
 
 def get_protocol(state: State) -> str:
```

Comment:
This code is old and should be removed, please here is the master's branch code
https://github.com/CarmineOptions/derisk-research/blob/master/apps/dashboard_app/helpers/loans_table.py#L10